app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,6 +12,6 @@ class VerifyCsrfToken extends BaseVerifier
      * @var array
      */
     protected $except = [
-        //
+        'import/email-subscription',
```

Comment:
Added this here in order to test with `$this->postJson` test method. I was receieved a `419` status code otherwise and found this suggestion [here](https://stackoverflow.com/questions/46266553/why-laravel-api-return-419-status-code-on-post-and-put-method).